client/src/types/types.d.ts

```diff
@@ -0,0 +1,12 @@
+//TODO: figure out why this file isn't parsing in VSCode but still successfully prevents type errors when importing yaml
+interface TextBundle {
```

Comment:
Yaml and CSV imports all follow these types. It's not perfect, we'll probably have more work to do once we convert `text.js`